### PR TITLE
add some documentation for setting up Helm development

### PR DESCRIPTION
**Pull request by RangerRick**
This PR adds some stuff for helping to get set up for development.

### CODE REVIEW

File: docs/modules/contributing/pages/developing.adoc [hunk 1]
```diff
@@ -1,7 +1,46 @@
...
-Commits to the `develop` branch are automatically compiled and committed to the `master` branch.
+Development is done in the https://github.com/opennms/opennms-helm/tree/develop[`develop`] (default GitHub) branch.
+
+Commits to the `develop` branch are automatically compiled, tested, and merged to the https://github.com/opennms/opennms-helm/tree/develop[`master`] branch.
 
 Releases will be tagged from the `master` branch and submitted to the https://github.com/grafana/grafana-plugin-repository[Grafana plugin repository].
+
+## Setting Up Helm and Grafana for Development
+
+1. Install https://nodejs.org/en/download/[Node.js] and https://grafana.com/docs/grafana/latest/installation/[Grafana] for your platform
```
Comment (Bonrob2) [inline docs/modules/contributing/pages/developing.adoc:13]:
```suggestion
1. Install https://nodejs.org/en/download/[Node.js] and https://grafana.com/docs/grafana/latest/installation/[Grafana] for your platform.
```

File: docs/modules/contributing/pages/developing.adoc [hunk 2]
```diff
@@ -1,7 +1,46 @@
...
++
+[source]
+----
+app_mode = development
+# further down in the file:
+[plugins]
+allow_loading_unsigned_plugins = opennms-helm-app
+----
+
+3. Check out https://github.com/OpenNMS/opennms-helm[the Helm git repository]:
```
Comment (Bonrob2) [inline docs/modules/contributing/pages/developing.adoc:25]:
```suggestion
3. Check out the https://github.com/OpenNMS/opennms-helm[Helm git repository]:
```

File: docs/modules/contributing/pages/developing.adoc [hunk 3]
```diff
@@ -1,7 +1,46 @@
...
++
+[source]
+----
+git clone https://github.com/OpenNMS/opennms-helm.git
+cd opennms-helm
+----
+
+4. Build Helm and symlink the `dist` directory in your Helm source tree into the Grafana plugin path (you will need to change these paths to match your system)
```
Comment (Bonrob2) [inline docs/modules/contributing/pages/developing.adoc:33]:
```suggestion
4. Build Helm and symlink the `dist` directory in your Helm source tree into the Grafana plugin path (you will need to change these paths to match your system).
```

File: docs/modules/contributing/pages/developing.adoc [hunk 4]
```diff
@@ -1,7 +1,46 @@
...
+[source]
+----
+# install Helm's build and runtime dependencies
+npm ci
+# do a one-time build to prime the dist/ directory
+npm run dev
+# symlink the dist/ directory into Grafana's plugin dir
+ln -s /path/to/opennms-helm/dist /var/lib/grafana/plugins/opennms-helm-app
+# run a continuous build that watches for changes
+# after you save a change to helm, just refresh Grafana to get the updates
```
Comment (Bonrob2) [inline docs/modules/contributing/pages/developing.adoc:44]:
```suggestion
# after you save a change to Helm, just refresh Grafana to get the updates
```

**Review by Bonrob2 — DISMISSED** [review]
Just a few minor nits. This is great to have.

**Review by aramosvizcarra — APPROVED** [review]
Looks great!. Worked perfectly with the only exception of a gitlab ssh certificate to setup i needed to setup on the first try. after that no issues.

**Pull request merged**